brainspace/gradient/kernels.py

```diff
@@ -24,7 +24,7 @@ def _build_kernel(x, kernel, gamma=None):
         return np.corrcoef(x)
 
     if kernel in {'cosine', 'normalized_angle'}:
-        x = 1 - squareform(pdist(x, metric='cosine'))
+            x = cosine_similarity(x)
```

Comment:
If you remove the extra indentation it would be good to go :)

```py
if kernel in {'cosine', 'normalized_angle'}:
    x = cosine_similarity(x) # remove extra indentation - has to be at the same level with the if below
    if kernel == 'normalized_angle':
        x = 1 - np.arccos(x, x)/np.pi
    return x
```